crates/graph/src/range/elem/elem_enum/arena.rs

```diff
@@ -125,12 +125,12 @@ impl RangeArenaLike<Elem<Concrete>> for RangeArena<Elem<Concrete>> {
             self.map.insert(Elem::Null, 0);
         }
 
-        let nulls = self.ranges.iter().fold(0, |mut acc, e| {
-            if matches!(e, Elem::Null) {
-                acc += 1;
-            }
-            acc
-        });
+        // let nulls = self.ranges.iter().fold(0, |mut acc, e| {
+        //     if matches!(e, Elem::Null) {
+        //         acc += 1;
+        //     }
+        //     acc
+        // });
 
```

Comment:
dont know the original intention here, but decided to comment it rather than leading underscore the unused `nulls` var